webapp/components/LocaleSwitch/LocaleSwitch.vue

```diff
@@ -64,15 +66,44 @@ export default {
       })
       return routes
     },
+    ...mapGetters({
+      currentUser: 'auth/user',
+    }),
   },
   methods: {
     changeLanguage(locale, toggleMenu) {
       this.$i18n.set(locale)
+      this.change()
```

Comment:
I don't love this naming convention @ogerly, it doesn't tell me what the function does
shouldn't it be something like `updateUserLocale`?